icepyx/core/query.py

```diff
@@ -252,11 +281,50 @@ def end_time(self):
         """
         return self._end.strftime("%H:%M:%S")
 
+    @property
+    def cycles(self):
+        """
+        Return the unique ICESat-2 orbital cycle.
+
+        Examples
+        --------
+        >>> reg_a = icepyx.query.Query('ATL06',[-55, 68, -48, 71],['2019-02-20','2019-02-28'])
+        >>> reg_a.cycles
+        ['02']
+        """
+        return sorted(set(self._cycles))
+
+    @property
+    def tracks(self):
+        """
+        Return the unique ICESat-2 Reference Ground Tracks
+
+        Examples
+        --------
+        >>> reg_a = icepyx.query.Query('ATL06',[-55, 68, -48, 71],['2019-02-20','2019-02-28'])
+        >>> reg_a.tracks
+        ['0841', '0849', '0902', '0910']
+        """
+        return sorted(set(self._tracks))
+
+    @property
+    def orbit_number(self):
+        """
+        Return the ICESat-2 CMR orbit number
```

Comment:
for the test case where all tracks/cycles are used, no orbit number(s) are returned. Is this the expected behavior?

Reply:
yes that's the expected behavior as it'd be an unnecessary CMR parameter (basically would run as icepyx is now)